src/main/services/token/TransferTokenService.kt

```diff
@@ -35,4 +35,30 @@ object TransferTokenService {
             notes
         )
     }
+
+    fun execute(
```

Comment:
why do we need another execute?

Reply:
has a different signature

Reply:
could you just add token id to the other one?

Reply:
right, optional parameters :D :D :D

Reply:
FTW

Reply:
and extra nice is optional params where u set a default value and put those at the end of the method signature...makes calling the methods super clean!